spec/ride_spec.rb

```diff
@@ -0,0 +1,78 @@
+require './lib/ride'
+
+class Ride
+  attr_reader :name,
+              :distance,
+              :loop,
+              :terrain
+
+  def initialize(name, distance, loop, terrain)
```

Comment:
This initialize should live in the Ride class file (ride.rb) instead of the test file